product_to_bizerba_scale/models/product_scale_system_product_line.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2014-2017 GRAP (http://www.grap.coop)
+#   - Sylvain LE GAL (https://twitter.com/legalsylvain)
+# Copyright 2017-Today Coop IT Easy SCRLfs
+#   - Houssine BAKKALI <houssine@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+
+
+class ProductScaleSystemProductLine(models.Model):
+    _name = 'product.scale.system.product.line'
+    _description = 'Product Scale System Product Line'
+    _order = 'scale_system_id, sequence'
+
+    _TYPE_SELECTION = [
+        ('id', 'Product ID'),
+        ('numeric', 'Numeric Field'),
+        ('text', 'Char / Text Field'),
+        ('external_text', 'External Text Field'),
+        ('constant', 'Constant Value'),
+        ('external_constant', 'External Constant Text Value'),
+        ('many2one', 'Many2One Field'),
+        ('many2many', 'Many2Many Field'),
+        ('product_image', 'Product Image'),
+    ]
+
+    scale_system_id = fields.Many2one(
+        comodel_name='product.scale.system',
+        inverse_name='Scale System',
+        required=True,
+        ondelete='cascade',
+        index=True
+    )
+    company_id = fields.Many2one(
+        related='scale_system_id.company_id',
+        string='Company',
+        store=True
+    )
+    code = fields.Char(
+        string='Bizerba Code',
+        required=True
+    )
+    name = fields.Char(
+        string='Name',
+        required=True
+    )
+    sequence = fields.Integer(
+        string='Sequence',
+        required=True,
+        default=10
+    )
+    type = fields.Selection(
+        selection=_TYPE_SELECTION,
+        string='Type'
+    )
+    field_id = fields.Many2one(
+        comodel_name='ir.model.fields',
+        string='Product Field',
+        domain="[('model', 'in', ['product.template'])]"
+    )
+
+    # TODO Improve. Set domain, depending on the other field
```

Comment:
I suppose that this TODO comes from the 9.0 version.

Reply:
exactly :) I leave it